opencti-platform/opencti-graphql/tests/02-integration/01-database/filterGroup-test.js

```diff
@@ -1226,6 +1226,81 @@ describe('Complex filters combinations for elastic queries', () => {
       } });
     expect(queryResult.errors.length).toEqual(1);
   });
+  it('should list entities according to search filters with trunc word', async () => {
+    const queryResult = await queryAsAdmin({ query: REPORT_LIST_QUERY,
+      variables: {
+        first: 25,
+        filters: {
+          mode: 'and',
+          filters: [{
+            key: 'description',
+            // Look for this description value 'Report for testing purposes (random data).'
+            values: ['rt for testing purposes (rando'],
+            operator: 'search',
+            mode: 'or',
+          }],
+          filterGroups: [],
+        },
+      } });
+    expect(queryResult.data.reports.edges.length).toEqual(1);
+  });
+
+  it('should list entities according to search filters with trunc word and any order', async () => {
+    const queryResult = await queryAsAdmin({ query: REPORT_LIST_QUERY,
+      variables: {
+        first: 25,
+        filters: {
+          mode: 'and',
+          filters: [{
+            key: 'description',
+            // Look for this description value'Report for testing purposes (random data).'
+            values: ['for rt testing (rando purposes '],
+            operator: 'search',
+            mode: 'or',
+          }],
+          filterGroups: [],
+        },
+      } });
+    expect(queryResult.data.reports.edges.length).toEqual(1);
+  });
+
+  it('should not find entities according to search filters with trunc word and double quote', async () => {
+    const queryResult = await queryAsAdmin({ query: REPORT_LIST_QUERY,
+      variables: {
+        first: 25,
+        filters: {
+          mode: 'and',
+          filters: [{
+            key: 'description',
+            // Look for this description value'Report for testing purposes (random data).'
+            values: ['"rt for testing purposes (rando"'],
+            operator: 'search',
+            mode: 'or',
+          }],
+          filterGroups: [],
+        },
+      } });
+    expect(queryResult.data.reports.edges.length).toEqual(0);
+  });
+  it('should find entities according to search filters with double quote', async () => {
+    const queryResult = await queryAsAdmin({ query: REPORT_LIST_QUERY,
+      variables: {
+        first: 25,
+        filters: {
+          mode: 'and',
+          filters: [{
+            key: 'description',
+            // Look for this description value'Report for testing purposes (random data).'
+            values: ['"for testing purposes"'],
+            operator: 'search',
+            mode: 'or',
+          }],
+          filterGroups: [],
+        },
+      } });
+    expect(queryResult.data.reports.edges.length).toEqual(1);
+  });
+
```

Comment:
maybe add a test for
- a query with a filter like this: operator=search, key=description, values=[a_word_in_no_description] and we should find no entity 
- same query with 2 words in values: values=[a wo_word_in_no_description   a_word_in_an_entity_description] and we should find an entity